gossip/src/cluster_info.rs

```diff
@@ -92,9 +92,6 @@ use {
     },
 };
 
-pub const VALIDATOR_PORT_RANGE: PortRange = (8000, 10_000);
-pub const MINIMUM_VALIDATOR_PORT_RANGE_WIDTH: u16 = 12; // VALIDATOR_PORT_RANGE must be at least this wide
```

Comment:
let's re-export these from net-utils with a deprecation warning

Reply:
Agreed that it's a good idea, but re-exporting from net-utils seems to introduce a cyclic dependency since we would have to add the gossip dependency to net-utils.

Reply:
Sorry I meant retain the `net-utils` source instance and re-export from here.  We can just change the `use` statement in this file to `pub use` for these two symbols in this file